wait/host.go

```diff
@@ -126,6 +129,33 @@ func (a *HostAwaitility) WaitForUserSignup(name string, criteria ...UserSignupWa
 	return userSignup, err
 }
 
+// WaitForBannedUser waits until there is a BannedUser available with the given email
+func (a *HostAwaitility) WaitForBannedUser(email string) (bannedUser *toolchainv1alpha1.BannedUser, err error) {
+	md5hash := md5.New()
+	// Ignore the error, as this implementation cannot return one
+	_, _ = md5hash.Write([]byte(email))
+	emailHash := hex.EncodeToString(md5hash.Sum(nil))
+	labels := map[string]string{toolchainv1alpha1.BannedUserEmailHashLabelKey: emailHash}
```

Comment:
wouldn't it be good to extract this part into a separated function that would be called here as well as at these two places?:
https://github.com/codeready-toolchain/toolchain-e2e/pull/80/files#diff-cc443128665ec194918207e25d44b3eeR173
https://github.com/codeready-toolchain/toolchain-e2e/pull/80/files#diff-4f14cf49a084497bcfac1b29c3bbf79bR204

Reply:
Good idea, I've added a function for this.